Library/Formula/gringo.rb

```diff
@@ -26,7 +25,12 @@ def install
     inreplace "SConstruct",
               "env['CXX']            = 'g++'",
               "env['CXX']            = '#{ENV.cxx}'"
-    scons "--build-dir=release", "gringo", "clingo"
-    bin.install "build/release/gringo", "build/release/clingo"
+    # Fix build problems
+    # https://sourceforge.net/p/potassco/bugs/104/
+    inreplace "libclasp/src/clasp_output.cpp",
+              "	using std::isnan;",
```

Comment:
I tried it without and it doesn't break anything. I could remove the spacing, if desired.